pkg/ovs/openflow/ofctrl_packetout.go

```diff
@@ -19,6 +19,8 @@ import (
 	"math/rand"
 	"net"
 
+	"k8s.io/klog"
+
```

Comment:
shouldn't be separated from "github.com/contiv"

Reply:
Fixed.